### ISSUE TITLE
Put ProgressBar in a Separate Package?

### ISSUE THREAD

**Issue by matthieugomez** [JuliaLang/Pkg.jl#2512]
The progress bar is great! It would be great to have it in a separate package, so that developers can use it in their own packages.

**Comment by fredrikekre** [comment]
What about https://github.com/cloud-oak/ProgressBars.jl, https://github.com/timholy/ProgressMeter.jl, https://github.com/JunoLab/ProgressLogging.jl ?

**Comment by matthieugomez** [comment]
I tried `ProgressMeter` but it did not work for me. It seems like that package is primarily made for computations that take hours (e.g. default printing is in 00:00:00). 

In contrast, the progressbar in `Pkg` is perfect for these computations that take 0-30sec (opening CSV, estimating model, etc). Moreover, every user is already familiar with this particular progressbar since it appears when updating packages. 

In the end, I ended up copying the code in my package [FixedEffects.jl(https://github.com/FixedEffects/FixedEffects.jl/blob/master/src/utils/progressbar.jl). So, it's fine for me, but I doubt I'm the only one interested in it!
